Styles/style.css

```diff
@@ -91,27 +91,39 @@ body {
   margin: 0.5rem auto;
   max-width: 42rem;
   background-color: #323335a6;
-  box-shadow: 0 8px 32px 0 rgba(59, 59, 59, 0.37);
   backdrop-filter: blur(1px);
   -webkit-backdrop-filter: blur(1px);
   border: 2px solid rgba(255, 255, 255, 0.082);
   background-color: transparent;
+  border: 1px solid #2bc48a;
 }
 
 .hljs {
   position: relative;
   font-family: "Source Code Pro", monospace;
   display: block;
-  /* overflow-x: auto; */
   padding: 1em 0.5em;
   background: transparent;
-  color: #f3cb5e;
+  color: #ff8f00;
   border-radius: 5px;
   line-height: 0.8;
   text-align: left;
-  border: 1px solid rgba(255, 255, 255, 0.082);
-  box-shadow: 4px 6px 3px #292e36, -4px -3px 4px #313640;
+  border: 1px solid rgb(180 176 176 / 71%);
+
+  background-color: #f6f7f9;
+}
+.social-links {
+  text-align: right;
+  position: absolute;
+  top: 1rem;
+  right: 7rem;
+  padding: 0.5em 1rem;
 }
+.social-links a {
+  font-size: large;
```

Comment:
Try to specify a CSS unit for sizes. What you did isn't wrong either, but using a proper unit is a good practice generally.